settings/memcache.settings.php

```diff
@@ -51,6 +60,10 @@
           'factory' => ['@memcache.backend.cache.factory', 'get'],
           'arguments' => ['container'],
         ],
+        'lock.container' => [
+          'class' => 'Drupal\memcache\Lock\MemcacheLockBackend',
+          'arguments' => ['container', '@memcache.backend.cache.container'],
+        ],
```

Comment:
So this means we're forcing locking into non-persistent store by default?
I've rarely come across a legitimate reason to put locking into memcache. Usually its an early indicator that an application is trying to do an atomic operation too regularly. Such operations have performance implications as too many requests have to wait to acquire the lock or let the atomic operation complete before continuing. This is why we see it as an "issue" so often. But really, its the symptom which this change would then hide.

Reply:
So your argument is to make the symptom -more pronounced-?